### ISSUE TITLE
Potential place the migrator should error out

### ISSUE THREAD

**Issue by nex3** [sass/migrator#119]
A user on twitter has [reported a case](https://twitter.com/bm_stschneider/status/1186166540925833216) where the migrator takes valid Sass and produces what seems to be invalid Sass. @jathak can you investigate? I think this might be because it used to be loading stuff into the global scope which it's not anymore.

**Comment by jathak** [comment]
Yeah, it looks like Bootstrap 3 assumes that its stylesheets are imported in a shared scope in a particular order, so it's not possible to just `@use` particular files.

Catching this is tricky, since we need to somehow detect when a stylesheet that is being migrated depends on stylesheets that aren't being migrated in an order-dependent way.

**Comment by nex3** [comment]
Could we combine the existing logic for detecting if a member is being used from a scope it can't import with the logic for detecting if a member needs to be renamed in an un-migrated dependency?
